cookbook/product_asset/sftp.rst

```diff
@@ -0,0 +1,94 @@
+How to connect to an external server for storage
+================================================
+
+Quick Overview
+--------------
+
+**This cookbook will demonstrate you how to store the assets on a SFTP instead of the default local filesystem. Assets are specific files available only in the Enterprise Edition. But you can apply the exact same cookbook to store your product medias elsewhere.**
+
+This cookbook assumes that you already configured an external server. This cookbook should be made before the installation of the PIM. Otherwise you will have to migrate your data to your external server.
+
+Connect to an SFTP server
+-------------------------
+
+Let's assume that you already have a fully functional SFTP server ready to receive files. You will have to setup the adapter in the ``app/config/config.yml`` file.
+
+.. code-block:: yaml
+
+    # /app/config/config.yml
+    oneup_flysystem:
+        adapters:
+            asset_storage_adapter:
+                SFTP:
+                    host: ftp.domain.com
+                    port: ~
+                    username: ~
+                    password: ~
+                    root: ~
+                    timeout: ~
+                    privateKey: ~
+                    permPrivate: ~
+                    permPublic: ~
+
+.. note::
+    Don't forget to add your SFTP credentials.
+    If you want to change the way your product media are stored, you have to configure the key XXX instead of asset_storage_adapter.
```

Comment:
put this in a different note

seriously ???
maybe XXX could be changed by the real key...
and code should be surrounded by `